core/pkg/ingress/controller/controller.go

```diff
@@ -676,6 +676,15 @@ func (ic *GenericController) getBackendServers() ([]*ingress.Backend, []*ingress
 
 // GetAuthCertificate ...
 func (ic GenericController) GetAuthCertificate(secretName string) (*resolver.AuthSSLCert, error) {
+	key, err := ic.GetSecret(secretName)
+	if err != nil {
+		return &resolver.AuthSSLCert{}, fmt.Errorf("unexpected error: %v", err)
+	}
+	if key != nil {
+		ic.secretQueue.Enqueue(key)
+	}
+	// Enough time to enqueue the new certificate
+	time.Sleep(5 * time.Second)
```

Comment:
@bprashanth Had some failure here because of this, but can't remember right now. So I'm removing this sleep to see what happens